images/win/scripts/SoftwareReport/SoftwareReport.Tools.psm1

```diff
@@ -260,3 +260,9 @@ function Get-AZDSVersion {
     $azdsVersion = $(azds --version) | Select-String "(\d+\.\d+\.\d+.\d+)"
     return "Azure Dev Spaces CLI $azdsVersion"
 }
+
+function Get-DacFxVersion {
```

Comment:
Could you please also add call of this function to https://github.com/actions/virtual-environments/blob/main/images/win/scripts/SoftwareReport/SoftwareReport.Generator.ps1